### PR TITLE
Feat/dbt init

### PR DESCRIPTION
**Pull request by mkryed**
- dbt init
- requirement.txt file added
- packages.yml file added
- revise .gitignore file

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# name or the intended use of these models
+name: 'dbt_demo_healthcare'
+version: '1.5.1'
+config-version: 2
+
+# This setting configures which "profile" dbt uses for this project.
+profile: 'dbt_demo_healthcare'
+
+# Require dbt 1.0.0+
+require-dbt-version: [">=1.3.0", "<1.5.2"]
```
Comment (90awlui) [inline dbt_project.yml:13]: what's the rationale for choosing these versions?

File: logs copy/dbt.log [hunk 2]
```diff
@@ -0,0 +1,57 @@
+[0m14:00:53.382239 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103514a00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1054d7b20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1054d7e80>]}
```
Comment (90awlui) [inline logs copy/dbt.log]: I'm not sure if the folder should be called `logs copy` or just `logs`. In the .gitignore it seems like this is the kind of file that would typically be ignored.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+dbt-bigquery==1.5.1
```
Comment (90awlui) [inline requirements.txt:1]: Do you need to also specify the version of dbt core or is that included with the bq connector?

In general, I think in the future we'll be moving away from using a requirements.txt and using something like poetry. You don't have to do anything about this in the present, just a heads up

File: dbt_project.yml (hunk 1 above)
Comment (mkryed) [reply dbt_project.yml:13]: Just to make sure dbt above a certain version is installed as we have done in poutine shop.

File: logs copy/dbt.log (hunk 2 above)
Comment (mkryed) [reply logs copy/dbt.log]: Correct! That is something that should was copied twice. I will push another code with this folder removed. Thanks

File: requirements.txt (hunk 3 above)
Comment (mkryed) [reply requirements.txt:1]: It is included.
Thanks for the heads up- We may as well create a ticket around how to implement poetry to replace the requirement.txt for this project and work on that on a later stage. We could all learn from it.

File: dbt_project.yml (hunk 1 above)
Comment (90awlui) [reply dbt_project.yml:13]: Sorry, should've clarified my question. Why do we choose 1.3.0 to be the minimum version of dbt? Is there something standout about that version?

**Review by 90awlui — APPROVED** [review]

**Pull request merged**